NitroxServer/GameLogic/Players/PersistedPlayerData.cs

```diff
@@ -40,6 +41,9 @@ public class PersistedPlayerData
         [JsonProperty, ProtoMember(10)]
         public bool IsPermaDeath { get; set; }
 
+        [JsonProperty, ProtoMember(11)]
+        public Guid Token { get; set; }
```

Comment:
is this type appropriately handled by both serializers?

Reply:
I did not test the PROTOBUF serializer scenario, only the JSON one.

Reply:
Pls add this field to the `WorldPersistentTest` to check if it is working with ProtoBuf.